healthworkforce/README.md

```diff
@@ -0,0 +1,56 @@
+# Setup
+
+## Kubernetes
+
+Navigate to the folder `/kubernetes/main` to execute the commands.
+
+To set up the healthworkforce services run the following command:
+
+```sh
+./k8s.sh init
+```
```

Comment:
I dont think there is an init script for the hfw package? this was only needed in the core package?

Reply:
addressed in https://github.com/openhie/instant/pull/45/commits/ac4a7df3cba21c52d89937795d33e9cc349da517